models/articles.model.js

```diff
@@ -7,6 +7,33 @@ return db.query(`SELECT * FROM articles WHERE article_id = $1;`, [article_id])
     
 }
 
+exports.getAllArticlesObj = () => {
+    return db.query(`SELECT articles.author,
+                    articles.title, 
+                    articles.article_id,
+                    articles.topic,
+                    articles.created_at,
+                    articles.votes,
+                    articles.article_img_url,
+                    CAST(COUNT(comments.article_id) AS integer) AS comment_count 
+                    FROM articles
+                    LEFT JOIN comments
+                    ON comments.article_id = articles.article_id
+                    GROUP BY articles.author,
+                    articles.title, 
+                    articles.article_id,
+                    articles.topic,
+                    articles.created_at,
+                    articles.votes,
+                    articles.article_img_url,
+                    comments.article_id
+                    ORDER BY articles.article_id DESC;`)
```

Comment:
This is a good query but is a little larger than it needs to be, the GROUP BY does not need to contain so many column names for this to work.